sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -41,6 +41,7 @@ export type AnalyzeSentimentErrorResult = TextAnalyticsErrorResult;
 // @public
 export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
     includeOpinionMining?: boolean;
+    stringIndexType?: StringIndexType;
```

Comment:
not every API supports it as of right now, but perhaps in the future that would be a good idea once all of them have it (except for maybe just `beginAnalyze` because you can specify this option per task).

Reply:
I was talking with @suhas92 about it, and even though the encoding doesn't apply to every endpoint, the service doesn't fail or throw if it is specified.
We are considering on adding it to `TextAnalyticsOperationOptions` for .NET.